packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -4,21 +4,34 @@
 
 import FileSaver from 'file-saver';
 import { AppContext, AlertsContext } from '@substrate/ui-common';
-import { Balance, CopyButton, Dropdown, FadedText, Icon, Input, Margin, Menu, Modal, NavLink, Stacked, StackedHorizontal, StyledLinkButton, WithSpaceAround, WithSpaceBetween, SubHeader } from '@substrate/ui-components';
+import uiSettings from '@polkadot/ui-settings';
```

Comment:
What did we decide on import order? By default, I'm assuming it's still the one in the contributing guide? The only requirement I have is consistency, which isn't respected here.